examples/Microsoft.CognitiveServices_accounts_connections@2025-06-01/main.tf

```diff
@@ -0,0 +1,307 @@
+      allowedFqdnList = [
+      ]
+      apiProperties = {
+      }
+      disableLocalAuth              = false
+      dynamicThrottlingEnabled      = false
+      publicNetworkAccess           = "Enabled"
+      restrictOutboundNetworkAccess = false
+    }
+    sku = {
+      name = "S0"
+      tier = "Standard"
+    }
+  }
+  schema_validation_enabled = false
+  response_export_values    = ["*"]
+}
+
+resource "azapi_resource" "account_openai" {
+  type      = "Microsoft.CognitiveServices/accounts@2025-06-01"
+  parent_id = azapi_resource.resourceGroup.id
+  name      = "${var.resource_name}-openai"
+  location  = var.location
+  identity {
+    type = "SystemAssigned"
+  }
+
+  body = {
+    kind = "OpenAI"
+    properties = {
+      allowProjectManagement = true
+      allowedFqdnList = [
+      ]
+      apiProperties = {
+      }
+      disableLocalAuth              = false
+      dynamicThrottlingEnabled      = false
+      publicNetworkAccess           = "Enabled"
+      restrictOutboundNetworkAccess = false
+    }
+    sku = {
+      name = "S0"
+      tier = "Standard"
+    }
+  }
+  schema_validation_enabled = false
+  response_export_values    = ["*"]
+}
+
+resource "azapi_resource" "storageAccount" {
+  type      = "Microsoft.Storage/storageAccounts@2021-09-01"
+  parent_id = azapi_resource.resourceGroup.id
+  name      = var.resource_name
+  location  = var.location
+  body = {
+    kind = "StorageV2"
+    properties = {
+      accessTier                   = "Hot"
+      allowBlobPublicAccess        = false
+      allowCrossTenantReplication  = true
+      allowSharedKeyAccess         = false
+      defaultToOAuthAuthentication = false
+      encryption = {
+        keySource = "Microsoft.Storage"
+        services = {
+          queue = {
+            keyType = "Service"
+          }
+          table = {
+            keyType = "Service"
+          }
+        }
+      }
+      isHnsEnabled      = false
+      isNfsV3Enabled    = false
+      isSftpEnabled     = false
+      minimumTlsVersion = "TLS1_2"
+      networkAcls = {
+        bypass        = "AzureServices"
+        defaultAction = "Deny"
+        resourceAccessRules = [
+          {
+            resourceId = azapi_resource.account.id
+            tenantId   = data.azapi_client_config.current.tenant_id
+          }
+        ]
+      }
+      publicNetworkAccess      = "Enabled"
+      supportsHttpsTrafficOnly = true
+    }
+    sku = {
+      name = "Standard_LRS"
+    }
+  }
+  schema_validation_enabled = false
+  response_export_values    = ["*"]
+}
+
+resource "azapi_resource" "container" {
+  type      = "Microsoft.Storage/storageAccounts/blobServices/containers@2024-01-01"
+  parent_id = "${azapi_resource.storageAccount.id}/blobServices/default"
+  name      = var.resource_name
+  body = {
+    properties = {
+    }
+  }
+
+  schema_validation_enabled = false
+  response_export_values    = ["*"]
+  depends_on                = [azapi_resource.storageAccount]
```

Comment:
Please remove this as it's not necessary.

Reply:
fixed